docs/community.rst

```diff
@@ -3,13 +3,83 @@ Aiven community
 
 There are lots of other ways to learn more about Aiven, and interact with us.
 
-**Blog**: https://aiven.io/blog - to read tech news, tutorials, and updates on what we're up to.
 
-**Aiven Console**: https://console.aiven.io - your Aiven web interface. New accounts get a free trial.
+.. panels::
+    :card: shadow
 
-**Help articles**: https://help.aiven.io - more tech facing goodness to help you get things done.
+    🐦 **Twitter** We love to chat! Tell us what you're building with Aiven!
 
-**Twitter**: https://twitter.com/aiven_io - we love to chat! Tell us what you're building with Aiven :)
+    +++
 
-**YouTube**: https://www.youtube.com/c/Aiven_io - if video is your thing, cool, it's ours too.
+    .. link-button:: https://twitter.com/aiven_io
+        :type: ref
+        :text: To the Twitterverse
+        :classes: stretched-link
 
+    ---
+
+    📖 **Blog** To read tech news, tutorials, and updates on what we're up to.
+
+    +++
+
+    .. link-button:: https://aiven.io/blog
+        :type: ref
+        :text: To the blog
+        :classes: stretched-link
+
+    ---
+
+    📺 **YouTube** If video is your thing, cool, it's ours too.
+
+
+    +++
+
+    .. link-button:: https://www.youtube.com/c/Aiven_io
+        :type: ref
+        :text: To YouTube
+        :classes: stretched-link
+
+    ---
+
+    🐱 **GitHub** Find our public repositories on GitHub, work on `open source <https://aiven.io/open-source>`_ with us! 
+
+    +++
+
+    .. link-button:: https://github.com/aiven
+        :type: ref
+        :text: To the code
+        :classes: stretched-link
+
+
+Check out the `Aiven Console <https://console.aiven.io>`_ for your Aiven web interface. New accounts get a free trial!
+
+Say hi! 
+-------
+
+Meet us at the following events: 
+
+------------
+
+January 20, `PHP User Group Frankfurt <https://www.phpugffm.de/veranstaltungen-2/phpugffm-i-2022/>`_
+
+Join Lorna Jane for "Aiven for Laravel" 
+
+------------
+
+February 5-6, `FOSDEM <https://fosdem.org/2022/>`_
+
+Join Francesco Tisiot and Olena Kutsenko for their sessions 
```

Comment:
instead of names, I would suggest to add the github handle

Reply:
Or both? Since the casual reader will not necessarily realise who the github id means.

Reply:
Why not the Twitter handle?

Reply:
I think is like, most likely people want to contact you somehow and from github handle is more like looking what you are doing and github people can drop your message. Another option, it is e-mail. Up to the team. I'd be fine either way. :)

Reply:
And yes, comparing that to what is there now, that is much nicer.